main.go

```diff
@@ -41,9 +47,30 @@ func configure() *exporter.Config {
 	flag.DurationVar(&config.LivenessTimeout, "liveness-timeout", 10*time.Minute, "when should the service be considered un-live")
 	flag.DurationVar(&config.Welpenschutz, "health-welpenschutz", 10*time.Minute, "how long initially the service is considered healthy.")
 	flag.DurationVar(&config.DirectoryTimeout, "directory-timeout", 10*time.Minute, "how long to wait for missing directories")
+	flag.StringVar(&config.LogLevel, "log-level", "error", "set log level (debug, info, warn, error)")
+	flag.BoolVar(&config.LogJson, "log-json", false, "enable JSON-formatted logging")
 	flag.Parse()
+
+	switch config.LogLevel {
+	case "debug":
+		log.Level = logrus.DebugLevel
+	case "info":
+		log.Level = logrus.InfoLevel
+	case "warn":
+		log.Level = logrus.WarnLevel
+	case "error":
+		log.Level = logrus.ErrorLevel
+	default:
+		log.Fatalf("Unknown log level %s", config.LogLevel)
+	}
```

Comment:
That are a lot of levels. :) Consider to reduce them to two; for backward comparability it would be best to make debug the default.

Or just a boolean "quiet" which is false by default.